app/openai_utils.py

```diff
@@ -12,4 +12,4 @@ def generate_headline(client, text: str, image_url: str) -> str:
         ],
         max_tokens=50
     )
-    return response.choices[0].message['content'].strip()
+    return response.choices[0].message.content.strip()  # Corrected this line
```

Comment:
Remove this comment

Reply:
Sure, I'll remove the comment. Thank you for the feedback!

Reply:
Comment removed.

Reply:
Comment removed.

Reply:
Sure, I'll remove the comment from line 15 in the `app/openai_utils.py` file.

### Updated `app/openai_utils.py`

```python
from openai import OpenAI
import os

def generate_headline(client, text: str, image_url: str) -> str:
    prompt = f"Generate an ad headline for the following image URL based on the website text:\n\nWebsite Text:\n{text}\n\nImage URL: {image_url}\n\nAd Headline:"
    response = client.chat.completions.create(
        model="gpt-3.5-turbo",
        messages=[
            {"role": "system", "content": "You are a helpful assistant."},
            {"role": "user", "content": prompt}
        ],
        max_tokens=50
    )
    return response.choices[0].message.content.strip()
```

I'll go ahead and make this change.